frontend/src/components/navigation/Header.test.tsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
 import renderer from 'react-test-renderer';
-
-jest.mock('@react-keycloak/web');
 import { useKeycloak } from '@react-keycloak/web';
 import Header from './Header';
 
+jest.mock('@react-keycloak/web');
+
```

Comment:
I didn't do this. I have no idea if it impacts the tests...

Reply:
I don't think the order matters here.